frontend/src/metabase/public/containers/PublicDashboard/PublicDashboard.tsx

```diff
@@ -295,10 +300,12 @@ function isSuccessfulFetchDashboardResult(
   return !hasError;
 }
 
-export const PublicDashboard = _.compose(
-  connector,
+export const PublicDashboard = connector(PublicDashboardInner);
```

Comment:
The `PublicDashboardControlled` is the one used in iframe/embedding/public dashboard contexts that use the URL, and `PublicDashboard` is simply the raw dashboard with redux props.

Reply:
That is great context! Would you mind adding that to be the doc comment for `PublicDashboardControlled`, so other folks know this too? It's not immediately clear from just reading the PublicDashboardControlled's declaration.

Reply:
Added :smile:  I couldn't think of a different name either tbh